docs/renovate/readme.md

```diff
@@ -60,3 +60,102 @@ module.exports = {
   ]
 };
 ```
+
+### Complex case with private registries
+
+If some of your dependencies are coming from a private registry, you will have to add the token in the `.npmrc` or `.yarnrc.yml` to be able to fetch the new versions and trigger the install in postUpgradeCommands.
```

Comment:
```suggestion
If some of your dependencies are coming from a private registry, you will have to add the token in the `.npmrc` or `.yarnrc.yml` to be able to fetch the new versions and trigger the install in `allowedPostUpgradeCommands`.
```

Reply:
I am really talking about post-upgrade commands here, I've updated to clarify: `the post-upgrade commands from the presets`